tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -390,8 +412,15 @@ def test_show_priority_group_watermark_shared(self, setup, setup_config_mode):
             for intf in setup['up_ports']:
                 assert re.search(r'{}.*'.format(intf), show_pg) is not None
 
+
 class TestShowQueue():
 
+    @pytest.fixture(scope="class", autouse=True)
```

Comment:
Looks same as above. Define it outside the class?

Reply:
The fixture must be defined in the class to skip all the tests in the class. The logic to check if the tests need to be skipped is moved to common function